Makefile

```diff
@@ -56,6 +56,7 @@ build-env: # Make a new conda environment
 install: # Install setuptools, install this package in editable mode
 	pip install --upgrade pip setuptools
 	pip install -e .[DEV]
+	pip install git+https://github.com/ihmeuw/vivarium@${GIT_BRANCH}
```

Comment:
What if that upstream branch doesn't exist? Won't this fail?

Reply:
Yes. Isn't that what we want? That is the current behavior on github actions